.github/workflows/dapr-bot-schedule.yml

```diff
@@ -0,0 +1,61 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation and Dapr Contributors.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: dapr-automerge
+
+on:
+  schedule:
+    - cron: '*/10 * * * *'
+  workflow_dispatch:
+jobs:
+  automerge:
+    if: github.repository_owner == 'dapr'
+    name: Automerge and update PRs.
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: pip install PyGithub
+      - name: Automerge and update
+        env:
+          MAINTAINERS: yaron2,youngbupark,artursouza
+          GITHUB_TOKEN: ${{ secrets.DAPR_BOT_TOKEN }}
+        run: python ./.github/scripts/automerge.py
+  prune_stale:
+    name: Prune Stale
+    runs-on: ubuntu-latest
+    steps:
+    - name: Prune Stale
+      uses: actions/stale@v3.0.14
+      with:
+        repo-token: ${{ secrets.DAPR_BOT_TOKEN }}
+        # Different amounts of days for issues/PRs are not currently supported but there is a PR
+        # open for it: https://github.com/actions/stale/issues/214
+        days-before-stale: 30
+        days-before-close: 7
+        stale-issue-message: >
+          This issue has been automatically marked as stale because it has not had activity in the
+          last 30 days. It will be closed in the next 7 days unless it is tagged (pinned, good first issue, help wanted or triaged/resolved) or other activity
+          occurs. Thank you for your contributions.
+        close-issue-message: >
+          This issue has been automatically closed because it has not had activity in the
+          last 37 days. If this issue is still valid, please ping a maintainer and ask them to label it as pinned, good first issue, help wanted or triaged/resolved.
```

Comment:
The messages are slightly different than in dapr/dapr#3248. Not a big deal. Just calling it out.

Reply:
Since it has been updated by https://github.com/dapr/dapr/pull/3347.